# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../cups/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Imported from Fedora Core.

  - Rene Rebe <rene@exactcode.de>

--- cups-1.1.23/pdftops/GfxFont.cxx.gcc4	2005-02-18 14:04:54.252480058 +0000
+++ cups-1.1.23/pdftops/GfxFont.cxx	2005-02-18 14:05:28.154003304 +0000
@@ -969,7 +969,7 @@
   if ((nameToGID = ff->getNameToGID())) {
     for (i = 0; i < 256; ++i) {
       if (!map[i] && (charName = enc[i])) {
-	map[i] = (Gushort)(int)nameToGID->lookup(charName);
+	map[i] = (Gushort)(long)nameToGID->lookup(charName);
       }
     }
     delete nameToGID;
